tools/packaging/helm-chart/kata-deploy/templates/kata-deploy.yaml

```diff
@@ -77,6 +77,19 @@ spec:
           name: kata-artifacts
         - mountPath: /usr/local/bin/
           name: local-bin
+{{- with .Values.kataDeploy.kataOps }}
```

Comment:
do we need privileged: true security context in this container's spec?

Reply:
I tried this and it works fine without privileged.